src/main/java/Command.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Arrays;
+
+public class Command {
+    protected String command;
+    protected int argCount;
+    protected String[] arguments;
+
```

Comment:
maybe there should be a comment that explains what arguments string is to improve readability?